lua/svartafanan/window.lua

```diff
@@ -34,68 +37,60 @@ function M.open()
 	local row = math.ceil((height - win_height) / 2 - 1)
 	local col = math.ceil((width - win_width) / 2)
 
-	local border_opts = {
-		style = "minimal",
-		relative = "editor",
-		width = win_width + 2,
-		height = win_height + 2,
-		row = row - 1,
-		col = col - 1,
-	}
-
-	local opts = {
+	M.win = api.nvim_open_win(M.buf, true, {
 		style = "minimal",
 		relative = "editor",
 		width = win_width,
 		height = win_height,
 		row = row,
 		col = col,
-	}
+		border = "double",
```

Comment:
This creates the exact same thing as the border_opts I sent in? Is this new? Did I just do too much work for no purpose?